code/SerialPortStream.cs

```diff
@@ -283,16 +295,20 @@ public bool IsOpen
             }
         }
 
-        private object m_CloseLock = new object();
-
+        private object m_CloseLock = new object();
+
         /// <summary>
         /// Closes the port connection, sets the IsOpen property to false. Does not dispose the object.
         /// </summary>
         /// <remarks>
         /// This method will clean up the object so far as to close the port. Internal buffers remain
         /// active that the stream can continue to read. Writes will throw an exception.
         /// </remarks>
+#if !NETSTANDARD15
         public new void Close()
+#else
+        public void Close()
```

Comment:
Don't worry - my recompile wasn't right. .NET Standard doesn't have this method. https://docs.microsoft.com/en-us/dotnet/core/api/system.io.stream